package.json

```diff
@@ -3,19 +3,19 @@
   "license": "Apache-2.0",
   "name": "@kyma-project/busola",
   "version": "0.0.1-rc.8",
+  "type": "module",
   "scripts": {
     "postinstall": "cd backend && npm install",
     "start": "concurrently -c cyan,green npm:busola npm:backend",
-    "busola": "npm run copy-themes; PORT=8080 BROWSER=none GENERATE_SOURCEMAP=false react-app-rewired start",
+    "busola": "npm run copy-themes; BROWSER=none GENERATE_SOURCEMAP=false vite",
     "backend": "cd backend && npm run start",
     "eslint-check": "eslint src/ backend/",
     "lint-check": "prettier --config ./.prettierrc --ignore-path ./.prettierignore --check '**/*.{ts,tsx,js,jsx,json,html,css,yaml,md}'",
     "lint-fix": "prettier --config ./.prettierrc --ignore-path ./.prettierignore --write '**/*.{ts,tsx,js,jsx,json,html,css,yaml,md}'",
     "lint-staged": "lint-staged --concurrent=false",
     "test": "react-app-rewired test",
```

Comment:
How are unit tests working when we are triggering this using `react-app-rewired`?
we want to get rid of this commands

Reply:
We decided to move our unit tests to Vitest, since Jest was really difficult and overcomplicated to configure in this setup. That will be done in another PR for better readability